frontend/src/components/dashboard/create/common/BodyTemplate/BodyTemplate.tsx

```diff
@@ -9,26 +8,37 @@ import styles from './BodyTemplate.module.scss'
 
 import { SMSCampaign, SMSProgress, TelegramProgress } from 'classes'
 import {
-  TextArea,
-  NextButton,
   ErrorBlock,
+  NextButton,
   StepHeader,
   StepSection,
+  TextArea,
 } from 'components/common'
 import SaveDraftModal from 'components/dashboard/create/save-draft-modal'
 import { CampaignContext } from 'contexts/campaign.context'
 import { FinishLaterModalContext } from 'contexts/finish-later.modal.context'
 
-const SmsMessageBodyInfo = ({ body }: { body: string }) => {
-  const COST_PER_TWILIO_SMS_SEGMENT_IN_SGD = 0.0395 // correct as at 5 Feb 2022
+// correct as at 12 Jun 2022
+export const COST_PER_SMS = 0.0395
+export const USD_SGD_EXCHANGE_RATE = 1.4
```

Comment:
Can we return the result in SGD? the conversion should be done in the backend either:
- Using a configurable backend env var to set the exchange rate
- Making an API call to somewhere to retrieve live exchange rate

Reply:
that makes sense, will change!